content/news/articles/2025-04-17-opensearch-alerting.md

```diff
@@ -0,0 +1,71 @@
+---
+layout: post
+title: "OpenSearch Alerting"
+date: April 17th, 2025
+excerpt: How to setup alerting for your applications on Cloud.gov using OpenSearch
+---
+
+Opensearch Alerting has been added to Cloud.Gov opensearch, this new feature allows for users to monitor data and send alert when certain conditions or thresholds are met.
+
+You will be able to setup monitors for different conditions using fields, this will allow you to do things such as
+
+1. Track the metrics of your logs
+2. Query based on field specific filters (size of value, contents of value,etc)
+3. Write a query for a specific section of your logs in which you want to look for conditions to trigger on. (Alert on prod space only, This app only, etc)
+
+This will allow to see if your logs have a increase in 404 messages in a certain timeframe, track log contents for certain words, etc..
+
+## How access for OpenSearch notifications works
+
+Most of the objects that you create in OpenSearch, such as queries, visualization, or dashboards, are only visible to other members of your OpenSearch tenant. We have configured OpenSearch to have tenants for each cloud.gov organization, meaning there is a one-to-one mapping between tenants and organizations. For whichever tenant you choose when using OpenSearch, only other members of the associated cloud.gov organization can see your objects.
+
+Unfortunately, the OpenSearch alerting plugin [does not support the use of tenants for storing associated objects such as email recipient groups or channels](https://github.com/opensearch-project/alerting-dashboards-plugin/issues/1096).
+
+Given the current behavior of the OpenSearch alerting plugin, to ensure maximum security we have customized OpenSearch so that **only users who share all of the same cloud.gov organizations as your user will be able to see the notification objects that you create**. If another user shares some but not all of the same cloud.gov organizations as you, the names of your notification objects will be visible to the other user, but they will not be able to see any details about those objects.
+
+In OpenSearch 3.x, the concept of "workspaces" is going to be introduced, which may allow for a permissions model for notifications that more closely matches how we handle multi-tenancy for other OpenSearch objects.
+
+## Setting up a Notification
```

Comment:
The steps below are good, but I think it would be helpful to see some screenshots. Or, after the general instructions, do an example of creating an alert on high memory/CPU for an app and  include screenshots. That might be even better